spec/index.bs

```diff
@@ -3765,17 +3765,17 @@ interface GPUQueue {
     void signal(GPUFence fence, GPUFenceValue signalValue);
 
     void writeBuffer(
-        ArrayBuffer sourceData,
-        GPUSize64 sourceOffset,
-        GPUBuffer destination,
-        GPUSize64 destinationOffset,
-        GPUSize64 size);
+        GPUBuffer buffer,
+        GPUSize64 bufferOffset,
+        ArrayBuffer data,
+        optional GPUSize64 dataOffset = 0,
+        optional GPUSize64 size);
 
     void writeTexture(
-      ArrayBuffer sourceData,
-      GPUTextureDataLayout sourceLayout,
       GPUTextureCopyView destination,
-      GPUExtent3D size);
+      GPUExtent3D size,
```

Comment:
Should this `size` also be the last arg like in writeBuffer?

Reply:
No opinon, but it's true that it makes more sense now that the optional offset is removed.